config.go

```diff
@@ -32,6 +24,30 @@ type Config struct {
 	GitHubToken string `env:"REG_TOKEN" long:"token" short:"t" description:"Token used to connect to the API"`
 }
 
+// GitServerConfig holds the configuration for GitServer
+type GitServerConfig struct {
+	// GitServerPort is the port where the local git server will listen
+	GitServerPort int `env:"REG_GITPORT" default:"9418" long:"gitport" description:"Port for local git server"`
+	// RepositoriesFile
+	RepositoriesFile string `env:"REG_REPOS_FILE" default:"" long:"repos-file" description:"YAML file with the list of repos"`
+	// RepositoriesCache is the path to the downloaded repositories
+	RepositoriesCache string `env:"REG_REPOS" default:"repos" long:"repos" description:"Directory to store repositories"`
+	// Complexity has the max number of complexity of repos to test
+	Complexity int `env:"REG_COMPLEXITY" default:"1" long:"complexity" short:"c" description:"Complexity of the repositories to test"`
+}
+
+// CIConfig is config that represents CI build metadata, values of CIConfig will be added to metrics as labels
+type CIConfig struct {
+	Branch string `env:"GIT_BRANCH" default:"" long:"ci-branch" description:"branch env"`
+	Commit string `env:"GIT_COMMIT" default:"" long:"ci-commit" description:"commit env"`
+}
```

Comment:
I believe this information is already passed as a parameter `remote:master` or `pr:345`. What's the use of these ones?

Reply:
they only used as labels for prometheus metrics